components/lib/dataprotect/src/main/java/mozilla/components/lib/dataprotect/KeyUtils.kt

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.dataprotect
+
+import java.security.SecureRandom
+
+/**
+ * @param keyStrength The strength of the generated key in bits
+ */
+@Suppress("MagicNumber")
+fun generateEncryptionKey(keyStrength: Int): String {
```

Comment:
nit: Does this need to be public? Is this something that app code is going to need to call? Such public functions without a scope are kind of noisy since they will show up in autocomplete everywhere.